system/security/LdapSecurity/ldaputils.cpp

```diff
@@ -231,9 +231,6 @@ LDAP* LdapUtils::ldapInitAndSimpleBind(const char* ldapserver, const char* userD
     *err = LdapSimpleBind(ld, timeout, (char*)userDN, (char*)pwd);
     if (*err != LDAP_SUCCESS)
     {
-        DBGLOG("LdapSimpleBind error (%d) - '%s' for admin user %s", *err, ldap_err2string(*err), isEmptyString(userDN) ? "NULL" : userDN);
-        if (!isEmptyString(userDN))
-            DBGLOG("Please make sure your LDAP configuration 'systemBasedn' contains the complete path, including the complete 'dc=domainComponent'");
```

Comment:
If not logged here, is it logged (or exception thrown) on the caller side?

Reply:
This behavior is unchanged. This routine logs the error, and the caller (CLdapConfig ctor) throws an exception (see ldapconnection.cpp, line 479)

        if(rc != LDAP_SUCCESS)
        {
            throw MakeStringException(-1, "getServerInfo error - %s", ldap_err2string(rc));
        }

Reply:
> This routine logs the error, .

I may misunderstand. If you remove those 3 lines, where do you log the error?

Reply:
I no longer log the error from the anonymous bind  (see PR comment #2)
     "Don't log ananymous bind errors, just quietly try with creds"
If the anonymous bind fails, I quietly retry with the provided creds. If that one fails , I log it on line 253
     DBGLOG("ldap bind error (%d) - %s", err, ldap_err2string(err));
It's up to the caller to throw or return the error, but in this case the caller throws